metabase/metabase
# Cleanup css modules usage

### Description

Cleanup CSS modules usage. To find unused places, I utilized https://github.com/bmatcuk/eslint-plugin-postcss-modules

it provides many false positives, so I'd not want to introduce this tool as a part of our flow, but it's helpful for manual cleanup

### How to verify

double check if all the removed classes are not used
double check if all the removed class usages are not defined 
verify percy
